atlasdb-ete-test-utils/src/main/java/com/palantir/atlasdb/todo/generated/SnapshotsStreamStore.java

```diff
@@ -179,22 +184,35 @@ protected File createTempFile(Long id) throws IOException {
     }
 
     @Override
-    protected void loadSingleBlockToOutputStream(Transaction t, Long streamId, long blockId, OutputStream os) {
-        SnapshotsStreamValueTable.SnapshotsStreamValueRow row = SnapshotsStreamValueTable.SnapshotsStreamValueRow.of(streamId, blockId);
+    protected void loadBlocksToOutputStream(Transaction t, Long streamId, long firstBlock, long numBlocks, OutputStream os) {
         try {
-            os.write(getBlock(t, row));
+            BiMap<SnapshotsStreamValueTable.SnapshotsStreamValueRow, Long> blockRows
+                    = KeyedStream.of(
+                    LongStream.rangeClosed(firstBlock, firstBlock + numBlocks).boxed())
+                    .mapKeys(blockId -> SnapshotsStreamValueTable.SnapshotsStreamValueRow.of(streamId, blockId))
+                    .collectTo(HashBiMap::create);
+            Map<SnapshotsStreamValueTable.SnapshotsStreamValueRow, byte[]> blocks = getBlocks(t, blockRows.keySet());
+            for (long i = 0; i < numBlocks; i++) {
+                os.write(blocks.get(blockRows.inverse().get(firstBlock + 1)));
```

Comment:
Uhh, this looks very much wrong - should this not be `firstBlock + i` like the rest?!

Reply:
Fixed.